data_steward/cdr_cleaner/cleaning_rules/deid/conflicting_hpo_state_generalization.py

```diff
@@ -0,0 +1,237 @@
+"""
+Cleaning rule to generalize conflicting HPO states
+"""
+import logging
+
+from google.cloud import bigquery
+
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule, query_spec_list
+from common import JINJA_ENV, OBSERVATION
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+
+LOGGER = logging.getLogger(__name__)
+
+JIRA_ISSUE_NUMBERS = ['DC512', 'DC834']
+JIRA_ISSUE_URL = [
+    'https://precisionmedicineinitiative.atlassian.net/browse/DC-512',
+    'https://precisionmedicineinitiative.atlassian.net/browse/DC-834'
+]
+
+MAP_TABLE_NAME = "person_src_hpos_ext"
+
+SCHEMA_MAP_TABLE = [{
+    "type": "integer",
+    "name": "person_id",
+    "mode": "required",
+    "description": "the person_id of someone with an ehr record"
+}, {
+    "type": "string",
+    "name": "src_id",
+    "mode": "required",
+    "description": "the src_id of an ehr record"
+}]
```

Comment:
We could potentially move this schema definition into its own json file in `data_steward/resource_files/schemas/internal/lookup_tables`.